.github/workflows/schema-spy.yml

```
@@ -0,0 +1,38 @@
+name: Schema Spy report generation
+
+on:
+  schedule:
+    - cron: "0 7 * * TUE" # Every tuesday at 07:00 UTC
```

Does this run against the delius-test environment?  If so, we should probably change this to later as the deployment Delius deployment usually happens around midday on a Tuesday.

Also is it worth running this whenever we push changes to the schema spy config?  To make sure it still works if we change config or version etc:
```suggestion
  schedule:
    - cron: "0 7 * * TUE" # Every tuesday at 07:00 UTC
  push:
    paths:
      - schema-spy/*
      - .github/workflows/schema-spy.yml
```